### ISSUE TITLE
[ plotting ] Empty plots are possible

### ISSUE THREAD

**Issue by kamaev** [moira-alert/moira#254]
It takes to:
- do not use buff.bytes() for NotificationPackage when Render() fails, use []byte{} instead
- [limits/resolveLimits] remove or refactor MinAndMaxOfTime to provide valid and different from/to's
- add tests for different retention intervals

**Comment by borovskyav** [comment]
[metric.txt](https://github.com/moira-alert/moira/files/2857315/metric.txt)

**Comment by kamaev** [comment]
Main goal is to avoid errors like **infinite {x,y}-range delta** from wcharczuk/go-chart

**Comment by gane5hvarma** [comment]
Hi i would like to work on this issue.is anyone working on this?

**Comment by beevee** [comment]
@gane5hvarma OK, you can take this task.

**Comment by srv-twry** [comment]
@beevee I'll be happy to take this one.

**Comment by beevee** [comment]
> @beevee I'll be happy to take this one.

Yeah, sure you can tackle this. But we don't assign tasks to new contributors anymore, since they disappear more often than not. Send us a pull request, and we'll review it.

**Comment by hv7214** [comment]
@srv-twry are you still on it ? If not, then i picking this up

**Comment by srv-twry** [comment]
Sure feel free. I've closed my PR
